### DISCUSSION TITLE
VPAT and HECVAT status?

### DISCUSSION THREAD

**Discussion by omachado-id** [djplaner/canvas-collections#286] [Q&A]
Hello,

I've been quite impressed (which I've mentioned before) with Canvas Collections.  So much so that I brought it to the attention of my supervisor at my university.  This person liked the results of my individual trial, and I asked if/how we can have our LMS/IT folk review its adoption.  However, I was asked to contact you regarding Canvas Collections' [VPAT (accessibility](https://adacompliancepros.com/what-is-the-vpat-compliance-and-how-to-get-vpat-certification/)) and [HECVAT (data security)](https://securityscorecard.com/blog/what-is-hecvat-and-why-is-it-important/) status.  Apparently, my institution will not adopt any third-party tools for our Canvas instance without this VPAT/HECVAT documentation.  

So my question (suggestion?) is whether these data security and accessibility compliance items are something that you plan on providing at some point, perhaps as part of your development roadmap, so that institutions like mine can have adopt Canvas Collections while adhering to internal standards?

I will continue to work with my individual copy, but unfortunately, the items I mentioned are part of any platform-wide adoption at my institution, so I hope that these are things that you can provide.  You have a great tool in Canvas Collections; I'd love it if we could adopt it.

**Comment by djplaner** [comment]
G'day,

Must admit these weren't on the agenda. As a very small open source project it may be challenging.

However, I've been vaguely aware of these through colleagues who are a little more advanced than I. Beyond the obvious advantage for institutional adoption, there's also some personal advantage in exploring these more.  

I've [added an issue](https://github.com/djplaner/canvas-collections/issues/288). I will explore more and get back to you.

**Comment by omachado-id** [comment]
Hi,
I appreciate your willingness to look at these issues.  I can't ask for more, as I can certainly imagine that, as a small project developer, your time and resources that are applicable to the project are limited.  So just knowing that you're interested in addressing these areas is an encouraging sign not just for me, but others who might be interested in CC.  I hope that you can achieve them

And not to belabor it, but yes, larger institutions will probably have some vetting process for enterprise level tool adoptions that will touch on security, accessibility, etc.  You've already built a good support site with helpful information that helped me and will certainly help others.  Thanks for your response!
